projects/plugins/crm/changelog/fix-crm-html-entities-for-special-characters-in-textareas

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: fixed
+Comment: Special characters in textarea fields (contacts, transactions, quotes) produce visible HTML entities  
```

Comment:
I think we need to remove the `comment:` part as well, otherwise the changelog tool thinks it's an internal comment that shouldn't be included in our changelog

```suggestion

Special characters in textarea fields (contacts, transactions, quotes) produce visible HTML entities

```